### PR TITLE
Add Git Hub Actions [PAGOPA-1039]

### PR DESCRIPTION
**Pull request by AngeloCaporaso**
#### List of Changes

- GitHub Actions were added
- Directory structure was updated
- Azure DevOps pipelines were updated @deprecated
- Docker configuration was updated

#### How Has This Been Tested?

- By running GHA and Azure DevOps pipelines in DEV and UAT environment

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/update_infra.yml
Comment (jacopocarlini) [inline .github/workflows/update_infra.yml:1]: do we want to move the [terraform apim](https://github.com/pagopa/template-java-spring-microservice/tree/main/infra) to this repo?

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+RUN mvn clean package -Dmaven.test.skip=true
+
+#
+# Package stage
+#
+FROM --platform=linux/amd64 adoptopenjdk/openjdk16:alpine as builder
+COPY --from=buildtime /build/target/*.jar application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+
```
Comment (jacopocarlini) [inline Dockerfile:16]: do we want to add APM agent too?

Comment (AngeloCaporaso) [reply Dockerfile:16]: Is it not already included in this [image](https://github.com/pagopa/docker-base-springboot-openjdk11/blob/eda9cbeb7a0466641e5f173952286958f7abf5b8/Dockerfile#L7C1-L7C1)?

File: .github/workflows/update_infra.yml
Comment (AngeloCaporaso) [reply .github/workflows/update_infra.yml:1]: Great idea. I would do it with a dedicated PR, what do you think?

**Review by jacopocarlini — APPROVED** [review]
🏆

**Pull request merged**